### DISCUSSION TITLE
path expansion (语法糖)不能识别 .with

### DISCUSSION THREAD

**Discussion by zjarles** [alibaba/GraphScope#3189] [Q&A] [answered]
g.V().out('1..2', 'register').with('PATH_OPT', 'ARBITRARY').with('RESULT_OPT', 'ALL_V') 报错：
SyntaxError: invalid syntax  红箭头指向with，请问这是什么原因呢？

**Comment by BingqingLyu** [comment]
Sorry that we can not reproduce the error. 
I tried on the demo graph and it outputs expected results:
![image](https://github.com/alibaba/GraphScope/assets/28559306/c061edd8-2702-45f2-98e4-ab3d75fb81d9)

Could you please provide more info for us to reproduce the error?

**Comment by zjarles** [comment]
您好，我用的是jupyter，不是gremlin客户端，调用的graphscope和gremlin_python的python模块包，正在用的graphscope模块包版本是v0.23.0


from gremlin_python.process.anonymous_traversal import traversal
from gremlin_python.driver.driver_remote_connection import DriverRemoteConnection
g = traversal().with_remote(DriverRemoteConnection('ws://127.0.0.1:8000/gremlin','g'))
interactive = graphscope.gremlin(graph)







------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "alibaba/GraphScope"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年9月6日(星期三) 晚上6:05
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [alibaba/GraphScope] path expansion (语法糖)不能识别 .with (Discussion #3189)





 
Sorry that we can not reproduce the error.
 I tried on the demo graph and it outputs expected result:
 
 
Could you please provide more info, e.g., the version of graphscope you used?

**Reply by zjarles** [reply, marked as answer]
需要用graphscope的客户端interactive.execute("...")来运行语法糖语句
